routes/hot_feed.go

```diff
@@ -517,6 +502,36 @@ func (fes *APIServer) PopulateHotnessInfoMap(
 	hotnessInfoMap := make(map[lib.BlockHash]*HotnessPostInfo)
 	// Map of interaction key to transaction type multiplier applied.
 	postInteractionMap := make(map[HotFeedInteractionKey]uint64)
+
+	// Fake block height for mempool transactions that haven't been mined yet
+	var mempoolBlockHeight int
+	if len(hotnessInfoBlocks) > 0 {
+		mempoolBlockHeight = hotnessInfoBlocks[0].BlockAge + 1
+	} else {
+		mempoolBlockHeight = 1
+	}
+
+	// Create new "block" for mempool txns, give it a block age of 1 greater than the current tip
+
+	// First get all MempoolTxns from mempool.
+	dbMempoolTxnsOrderedByTime, _, err := fes.backendServer.GetMempool().GetTransactionsOrderedByTimeAdded()
+	// Extract MsgDesoTxn from each MempoolTxn
+	var txnsFromMempoolOrderedByTime []*lib.MsgDeSoTxn
+	for _, mempoolTxn := range dbMempoolTxnsOrderedByTime {
+		txnsFromMempoolOrderedByTime = append(txnsFromMempoolOrderedByTime, mempoolTxn.Tx)
+	}
+
+	if err != nil {
+		glog.Errorf("Error getting mempool transactions: %v", err)
+	} else if len(txnsFromMempoolOrderedByTime) > 0 {
+		hotnessInfoBlocks = append(hotnessInfoBlocks, &HotnessInfoBlock{
+			Block: &lib.MsgDeSoBlock{
+				Txns: txnsFromMempoolOrderedByTime,
+			},
+			BlockAge: mempoolBlockHeight,
+		})
+	}
```

Comment:
I generally prefer to have my `if err != nil` immediately following the declaration of that error. can you refactor this chunk of code such that we catch the error in the next line of code following its declaration?